lib/shelley/cardano-wallet.cabal

```
@@ -124,6 +124,7 @@ library
       Cardano.Wallet.Shelley.BlockchainSource
       Cardano.Wallet.Shelley.Compatibility
       Cardano.Wallet.Shelley.Compatibility.Ledger
+      Cardano.Wallet.Shelley.Compatibility.Parser
```

```suggestion
      Cardano.Wallet.Shelley.Compatibility.ParserTx
```

(? See other comment)